internal/app/components/courses/courseRecords.go

```diff
@@ -13,6 +13,8 @@ type CourseRecord struct {
 	Id             CourseRecordId `json:"id"`
 	Grade          CourseGrade    `json:"grade"`
 	CompletionDate *time.Time     `json:"completion_date"`
+	// prerequisites are satisfied if true
```

Comment:
misleading comment imo, maybe something like "user specified this course was overridden so no need to check pre-requisites"

Reply:
true